spec/models/task_spec.rb

```diff
@@ -1,5 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "タスクモデルのバリデーションテスト" do
+    it "[有効]タスクが有効である事"  do
+      task = build(:task)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+    
+    it "[有効]異なるタイトルなら有効である事" do
+      create(:task)
+      task = build(:task)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+    
+    it "[無効]タスクにタイトルが無い事" do
+      task = build(:task,title: nil)
```

Comment:
スペース空けておいた方が見やすいので追加しておきましょう。
```suggestion
      task = build(:task, title: nil)
```

Reply:
c9989f429ab7cbce5fe599bb13248106376558ee
スペースを空けました。また他に箇所も同様の対応をしました。